### ISSUE TITLE
Support for arm-osx?

### ISSUE THREAD

**Issue by corneliusroemer** [pycompression/python-isal#141]
Do you plan to support arm-osx in the future?

**Comment by rhpvorderman** [comment]
There are multiple  blockers for this.
1. No arm-osx support in upstream: https://github.com/intel/isa-l. This should be fixed on the develop branch so a new release needs to come out first.
2. I have no means of testing out wheels that come out as arm-osx
3. I don't know how to build arm-osx wheels. Supposedly this can be handled with cibuildwheel. But I do wonder if this will also manage to build the proper isa-l for inclusion. As long as I have no method to test it I am hesitant to release it.

Basically this requires someone else to step in and enable support. I cannot support it myself and I am unwilling to buy the expensive hardware in order to do so.

**Comment by rhpvorderman** [comment]
Oh really? I also co-maintain xopen (hence the shared pycompression group). That should not be the case as the dependency is entirely optional. Can you make an issue at the xopen feedstock and ping me? Then I can look into it when I have the time.

**Comment by rhpvorderman** [comment]
As of yet, arm-osx seems to build on the latest dev branch. However, ISA-L need to release officially before that can be implemented.

**Comment by rhpvorderman** [comment]
I just merged a PR that patches the current version of ISA-L 2.31 with a few patches that enable compilation on OSX ARM64. Thanks a lot to @hmaarrfk for identifying the necessary commits.

**Comment by rhpvorderman** [comment]
Wheelbuilding is now enabled for arm-osx in this library and python-zlib-ng. Patches for Python 3.13 are also included. On july 30th python 3.13 will get its first release candidate and wheels build for 3.13 should be stable. I plan to release this library, including the macos arm64 wheels at that time.
